packages/request-client.js/src/http-data-access.ts

```diff
@@ -2,12 +2,20 @@ import { DataAccessTypes } from '@requestnetwork/types';
 import Utils from '@requestnetwork/utils';
 import axios, { AxiosRequestConfig } from 'axios';
 
+import { EventEmitter } from 'events';
+
 // Maximum number of retries to attempt when http requests to the Node fail
 const HTTP_REQUEST_MAX_RETRY = 3;
 
 // Delay between retry in ms
 const HTTP_REQUEST_RETRY_DELAY = 100;
 
+// Maximum number of retries to attempt when http requests to the Node fail
```

Comment:
I don't really what does it mean "getting the confirmation" compared to the http request, maybe this comment need more clarity